web-components/svelte-react.ts

```diff
@@ -1,4 +1,4 @@
-import React, { useRef, useEffect } from "react"
+import { createElement, useRef, useEffect, forwardRef, useCallback, type ForwardedRef, type PropsWithChildren } from "react"
```

Comment:
Brave Core isn't set up to use synthetic default imports and this was breaking things.